doc/source/release/1.12.0-notes.rst

```diff
@@ -273,6 +273,14 @@ Deprecated features
 - All wavelet functions have been deprecated, as PyWavelets provides suitable
   implementations; affected functions are: ``signal.{daub, qmf, cascade,
   morlet, morlet2, ricker, cwt}``
+- ``scipy.integrate.trapz``, ``scipy.integrate.cumtrapz``, and ``scipy.integrate.simps`` have
+  been deprecated in favour of `scipy.integrate.trapezoid`, `scipy.integrate.cumulative_trapezoid`,
+  and `scipy.integrate.simpson` respectively and will be removed in SciPy 1.14.
+- The deprecation of the ``tol`` argument of ``scipy.sparse.linalg.{gcrotmk,lgmres,minres,tfqmr}``
+  is now consistent with the other iterative solvers and will be removed in SciPy 1.14
+  in favour of ``rtol``. Further, the default value of ``atol`` for these functions is due
+  to change to ``0.0`` in SciPy 1.14.
```

Comment:
The deprecation of `tol` happened across the board for all* sparse solvers in 1.12, see #18488.

\* Of course, there's an [exception](https://github.com/scipy/scipy/issues/19704)... 😑